### PR TITLE
HTBHF-950 refactor address test fixtures

### PR DESCRIPTION
**Pull request by gethyn1**

### CODE REVIEW

File: api/src/test/java/uk/gov/dhsc/htbhf/hmrc/entity/HouseholdFactory.java [hunk 1]
```diff
@@ -2,16 +2,18 @@
...
                 .build()
-                .addAdult(anAdult("Homer", "Simpson", "QQ123456C"))
-                .addAdult(anAdult("Marge", "Simpson", "QQ123456D"))
-                .addChild(aChild("Bart", "Simpson", 48))
-                .addChild(aChild("Lisa", "Simpson", 24))
-                .addChild(aChild("Maggie", "Simpson", 6));
+                .addAdult(anAdult(HOMER_FORENAME, SIMPSONS_SURNAME, HOMER_NINO))
+                .addAdult(anAdult(MARGE_FORENAME, SIMPSONS_SURNAME, MARGE_NINO))
+                .addChild(aChild("Bart", SIMPSONS_SURNAME, 48))
+                .addChild(aChild("Lisa", SIMPSONS_SURNAME, 24))
```
Comment (mikeSmith08) [inline api/src/test/java/uk/gov/dhsc/htbhf/hmrc/entity/HouseholdFactory.java]: You have a `LISA_FORENAME` constant you can use.

File: api/src/test/java/uk/gov/dhsc/htbhf/hmrc/testhelper/TestConstants.java [hunk 2]
```diff
@@ -5,11 +5,27 @@
...
+    public static  final String SIMPSONS_ADDRESS_LINE_1 = "742 Evergreen Terrace";
+    public static  final String SIMPSONS_ADDRESS_LINE_2 = "Mystery Spot";
+    public static  final String SIMPSONS_TOWN_OR_CITY = "Springfield";
+    public static final String SIMPSONS_POSTCODE = "AA11AA";
+
     public static final LocalDate LISA_DOB = LocalDate.parse("1985-12-31");
     public static final String LISA_NINO = "EB123456C";
     public static final String LISA_FORENAME = "Lisa";
-    public static final String LISA_SURNAME = "Simpson";
+    public static final String LISA_SURNAME = SIMPSONS_SURNAME;
```
Comment (mikeSmith08) [inline api/src/test/java/uk/gov/dhsc/htbhf/hmrc/testhelper/TestConstants.java]: I'm not sure this really helps, can we not just use `SIMPSONS_SURNAME`?

File: api/src/test/java/uk/gov/dhsc/htbhf/hmrc/entity/HouseholdFactory.java (hunk 1 above)
Comment (gethyn1) [reply api/src/test/java/uk/gov/dhsc/htbhf/hmrc/entity/HouseholdFactory.java]: Updated to use `LISA_FORENAME`

File: api/src/test/java/uk/gov/dhsc/htbhf/hmrc/testhelper/TestConstants.java (hunk 2 above)
Comment (gethyn1) [reply api/src/test/java/uk/gov/dhsc/htbhf/hmrc/testhelper/TestConstants.java]: Removed `LISA_SURNAME` in favour of `SIMPSONS_SURNAME`

**Review by mikeSmith08 — APPROVED** [review]

**Review by YetAnotherMatt — APPROVED** [review]

**Pull request merged**